packages/sdk-codegen-scripts/src/exampleMiner.spec.ts

```diff
@@ -75,7 +75,7 @@ describe('example mining', () => {
     it('getRemoteHttpOrigin', () => {
       const actual = getRemoteHttpOrigin()
       expect(actual).toEqual(
-        'https://github.com/looker-open-source/sdk-codegen'
+        'https://github.com/looker-open-source/sdk-codegen.git'
```

Comment:
CI doesn't like this: [action output](https://github.com/looker-open-source/sdk-codegen/runs/5547364560?check_suite_focus=true)

Reply:
modified the test statement